centos7/cloudbuild.yaml

```diff
@@ -23,6 +23,13 @@ steps:
     id: "version"
     waitFor: ["-"]   # wait for nothing - start immediately
 
+# Step: pull the image used to validate YAML configs
+  - name: "l.gcr.io/google/bazel"
+    entrypoint: "docker"
+    args: ["pull", "gcr.io/asci-toolchain/container_release_tools/dependency_update/validators/semantics:latest"]
```

Comment:
Are we introducing a new dependency on a asci-toolchain container here?

Reply:
Discussed the reason for this offline